stratum/hal/bin/bcm/standalone/start-stratum.sh

```diff
@@ -17,30 +17,23 @@
 
 set -ex
 
-# Remove other SDK kernel modules, if present
-rmmod linux_bcm_knet || true
-rmmod linux_user_bde || true
-rmmod linux_kernel_bde || true
+load-kernel-modules.sh
 
-# Reinsert SDKLT kernel modules
-rmmod linux_ngknet || true
-rmmod linux_ngbde || true
-insmod linux_ngbde.ko && insmod linux_ngknet.ko
-sleep 1
+mkdir -p /var/run/stratum
 
 PLATFORM=$(cat /etc/onl/platform)
 
-./stratum_bcm \
+stratum_bcm \
     -external_stratum_urls=0.0.0.0:28000 \
-    -persistent_config_dir=/tmp/stratum \
-    -base_bcm_chassis_map_file=./stratum_configs/${PLATFORM}/base_bcm_chassis_map.pb.txt \
-    -chassis_config_file=./stratum_configs/${PLATFORM}/chassis_config.pb.txt \
-    -bcm_sdk_config_file=./stratum_configs/${PLATFORM}/SDKLT.yml \
-    -phal_config_path=./stratum_configs/${PLATFORM}/phal_config.pb.txt \
-    -bcm_serdes_db_proto_file=./stratum_configs/dummy_serdes_db.pb.txt \
-    -bcm_hardware_specs_file=./stratum_configs/bcm_hardware_specs.pb.txt \
-    -forwarding_pipeline_configs_file=/tmp/stratum/pipeline_cfg.pb.txt \
-    -write_req_log_file=/tmp/stratum/p4_writes.pb.txt \
+    -persistent_config_dir=/etc/stratum \
+    -base_bcm_chassis_map_file=/etc/stratum/stratum_configs/${PLATFORM}/base_bcm_chassis_map.pb.txt \
+    -chassis_config_file=/etc/stratum/stratum_configs/${PLATFORM}/chassis_config.pb.txt \
+    -bcm_sdk_config_file=/etc/stratum/stratum_configs/${PLATFORM}/SDKLT.yml \
+    -phal_config_path=/etc/stratum/stratum_configs/${PLATFORM}/phal_config.pb.txt \
+    -bcm_serdes_db_proto_file=/etc/stratum/stratum_configs/dummy_serdes_db.pb.txt \
+    -bcm_hardware_specs_file=/etc/stratum/stratum_configs/bcm_hardware_specs.pb.txt \
+    -forwarding_pipeline_configs_file=/var/run/stratum/pipeline_cfg.pb.txt \
+    -write_req_log_file=/var/run/stratum/p4_writes.pb.txt \
```

Comment:
I think we can use /var/log dir for logs.

Reply:
Yes, with this flag `-log_dir` for glog.